tools/src/faircv/Options.hs

```diff
@@ -0,0 +1,68 @@
+module Options
```

Comment:
Try not to use such general name, `stack ghci` goes mad when it sees two modules with the same name (except for `Main`).

Reply:
Done, I also renamed the Client module for the same reason.